### PR TITLE
Create State Pension page & homepage callouts

### PR DESCRIPTION
**Pull request by benbarnett**
State Pensions content needs to be populated before merging.

screen shot 2015-12-09 at 14 42 54

### CODE REVIEW

**Comment by benbarnett** [comment]
@andrewgarner look ok to you?

File: content/state_pension.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+If you reach State Pension age on or after 6 April 2016 you’ll get the [new State Pension](https://www.gov.uk/new-state-pension).
+
+## Get a State Pension statement
+
+You can get a [State Pension statement](https://www.gov.uk/state-pension-statement) to find out how much you may get and the number of qualifying years on your [National Insurance record](https://www.gov.uk/check-national-insurance-record).
+
+## Defer the State Pension
+
+You can increase your income in retirement if you don’t claim your State Pension when you reach State Pension age.This is called [deferring the State Pension](https://www.gov.uk/deferring-state-pension).
```
Comment (andrewgarner) [inline content/state_pension.md]: `age.This` is missing a space.

**Comment by andrewgarner** [comment]
We should add the new state pension guide to the examples in the [guides feature](https://github.com/guidance-guarantee-programme/pension_guidance/blob/master/features/guides.feature#L10).

File: content/state_pension.md (hunk 1 above)
Comment (benbarnett) [inline content/state_pension.md]: Fixed in https://github.com/guidance-guarantee-programme/pension_guidance/commit/b193560712b14db50c628bfd4966428fae97c55f

**Comment by benbarnett** [comment]
@andrewgarner guide added to examples added in https://github.com/guidance-guarantee-programme/pension_guidance/commit/b193560712b14db50c628bfd4966428fae97c55f

File: content/state_pension.md (hunk 1 above)
Comment (benbarnett) [inline content/state_pension.md]: Sorry this commit https://github.com/guidance-guarantee-programme/pension_guidance/commit/5acbf674c1663988b36ba716efd15b9093335fd7

**Comment by andrewgarner** [comment]
Are we going to change the [State Pension quick link](https://github.com/guidance-guarantee-programme/pension_guidance/blob/master/app/views/components/_quick_links.html.erb#L14)?

**Comment by benbarnett** [comment]
Have asked @alexkouz for his thoughts

**Comment by alexkouz** [comment]
I would change State Pension quick link to our new page.

**Comment by benbarnett** [comment]
Have updated in https://github.com/guidance-guarantee-programme/pension_guidance/commit/e95d16ca1832a36b1327edfa360a6a7d903ba450

**Comment by andrewgarner** [comment]
I've just checked the images using [ImageAlpha](https://pngmini.com) - I think we can reduce the file sizes. Does that sound like a good idea?

**Comment by benbarnett** [comment]
I've already run them through [ImageOptim](https://imageoptim.com/) which usually does a pretty good job.

**Comment by benbarnett** [comment]
Although yes worth it if we can save a few bytes.

**Pull request merged**